### PR TITLE
fix: update packages, add refresh token, cleanup

### LINKED ISSUE #5189: Review scope of zitadel-angular

### PR DESCRIPTION
**Pull request by peintnermax**
Closes https://github.com/zitadel/zitadel/issues/5189

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -13,69 +13,65 @@ import { StatehandlerService, StatehandlerServiceImpl } from './services/stateha
...
+    BrowserModule,
+    AppRoutingModule,
+    HttpClientModule,
+    OAuthModule.forRoot({
+      resourceServer: {
+        allowedUrls: [
+          '<YOUR DOMAIN>/caos.zitadel.auth.api.v1.AuthService',
+          '<YOUR DOMAIN>/oauth/v2/userinfo',
+          '<YOUR DOMAIN>/caos.zitadel.management.api.v1.ManagementService/',
+          'https://preview.api.zitadel.caos.ch',
```
Comment (eliobischof) [inline src/app/app.module.ts]:
```suggestion
```

**Review by eliobischof — COMMENTED** [review]
Please check for broken links.
Found one in the README.md:
image

`npm i` fails with npm version 9.6.2
image
`npm i --legacy-peer-deps` works

**Review by eliobischof — COMMENTED** [review]
Let's mention in the readme to configure the signout URL in the app

File: src/app/app.module.ts (hunk 1 above)
Comment (peintnermax) [reply src/app/app.module.ts]: done

**Comment by peintnermax** [comment]
> Let's mention in the readme to configure the signout URL in the app

Are you fine with linking to the setup in our docs? Its all described there and I'd keep this as clean as possible

File: src/app/app.module.ts (hunk 1 above)
Comment (peintnermax) [reply src/app/app.module.ts]: changed it to the correct api urls

**Comment by eliobischof** [comment]
> > Let's mention in the readme to configure the signout URL in the app
> 
> Are you fine with linking to the setup in our docs? Its all described there and I'd keep this as clean as possible

IMO, the problem is, that the app specific exact redirect URL value is nowhere mentioned.

**Comment by eliobischof** [comment]
do you have a solution for the `npm i` errors?

**Comment by peintnermax** [comment]
> > > Let's mention in the readme to configure the signout URL in the app
> > 
> > 
> > Are you fine with linking to the setup in our docs? Its all described there and I'd keep this as clean as possible
> 
> IMO, the problem is, that the app specific exact redirect URL value is nowhere mentioned.

I see.. will add it in https://github.com/zitadel/zitadel/pull/5534

**Review by eliobischof — APPROVED** [review]

**Comment by eliobischof** [comment]
is it a feat?

**Pull request merged**